README.md

```diff
@@ -1,3 +1,25 @@
+# Parser-Js
+
+### Installing
```

Comment:
Please, merge all of this new docs (or remove if repeated) into the previous README contents.

As you can see, and as @jonaslagoni mentioned, we don't want to have a worse documentation than what we have. It's about the format, the header of the repository, the TOC, etc. 
![Google Chrome_jjqAjmOb](https://github.com/asyncapi/parser-js/assets/1083296/be3a4d8a-9921-4f45-855a-afe208147b87)